.github/workflows/push_main.yaml

```diff
@@ -0,0 +1,14 @@
+name: Push (main)
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    uses: ./.github/workflows/build.yaml
+
+  publish:
+    needs: build
+    uses: ./.github/workflows/publish.yaml
```

Comment:
We are not testing before publishing, things could have changed between the CI run on the PR and the CI on main.

Reply:
Fixed